packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -849,7 +849,7 @@ class _StretchController extends ChangeNotifier {
     velocity = clampDouble(velocity, 1, 10000);
     _stretchSizeTween.begin = _stretchSize.value;
     _stretchSizeTween.end = math.min(_stretchIntensity + (_flingFriction / velocity), 1.0);
-    _stretchController.duration = Duration(milliseconds: (velocity * 0.02).round());
+    _stretchController.duration = Duration(milliseconds: math.max(velocity * 0.02, 1).round());
```

Comment:
I don't know what the fallback value should be here. 1 millisecond might look like jank. Also, maybe this duration should be exposed if folks want to customize it?

Reply:
Hello @Piinks,

It is a fair concern that it may look janky, although it looked ok in the devices that I used for testing (and simulators).

As an experiment, I set this animation to `const Duration(miliseconds: 1)` in the recording below

https://github.com/flutter/flutter/assets/82336674/f70ebfe4-271a-4473-b1d6-94eb1bb7b51e

Maybe it makes sense to make this minimum animation be `50ms`. 
This would match the shortest duration token in the material library, [Material 3 guidelines for easing and duration](https://m3.material.io/styles/motion/easing-and-duration/tokens-specs) (this token in the 
```suggestion
    _stretchController.duration = Duration(milliseconds: math.max(velocity * 0.02, 50).round());
```
This is what that would look like

https://github.com/flutter/flutter/assets/82336674/76efe488-5929-4d55-9303-bafe1cfa639c

What do you think?

Reply:
> Also, maybe this duration should be exposed if folks want to customize it

Good point, another issue can be raised for making this indicator more customisable overall, not only for this animation duration.
Potentially making use of material tokens as default values https://github.com/flutter/flutter/blob/bc88d0b9ed8c97550316b8e9ecdb7deae4529051/packages/flutter/lib/src/material/material.dart

 Maybe as part of #82906?

Reply:
Maybe! Are there tokens for the stretch effect now? 50 milliseconds SGTM

Reply:
I was not able to find a token for the stretch effect